libs/barista-components/copy-to-clipboard/src/copy-to-clipboard.ts

```diff
@@ -63,6 +67,17 @@ export class DtCopyToClipboard implements AfterContentInit, OnDestroy {
   /** Defines the button variant of the copy button. */
   @Input() variant: ButtonVariant = 'primary';
 
+  /** Defines if the copy button should be disabled */
+  @Input()
+  get disabled(): boolean {
+    return this._disabled;
+  }
+  set disabled(disabled: boolean) {
+    this._disabled = coerceBooleanProperty(disabled);
+    this._changeDetectorRef.markForCheck();
+  }
+  private _disabled: boolean = false;
```

Comment:
Please also add this, as it is needed for strict mode and coerced inputs.
` static ngAcceptInputType_disabled: BooleanInput;`

`BooleanInput` also comes from `'@angular/cdk/coercion';`

Reply:
Done, thanks